Density/app/src/main/java/org/cornelldti/density/density/facilitydetail/FacilityInfoPage.kt

```diff
@@ -137,44 +148,26 @@ class FacilityInfoPage : BaseActivity() {
     private fun setDayChipOnClickListener() {
         dayChips.setOnCheckedChangeListener { _, checkedId ->
             setDay(checkedId)
-            setDayMenu(checkedId)
         }
     }
 
     private fun setDay(checkedId: Int) {
-        var day = ""
-        when (checkedId) {
-            R.id.sun -> day = getString(R.string.SUN)
-            R.id.mon -> day = getString(R.string.MON)
-            R.id.tue -> day = getString(R.string.TUE)
-            R.id.wed -> day = getString(R.string.WED)
-            R.id.thu -> day = getString(R.string.THU)
-            R.id.fri -> day = getString(R.string.FRI)
-            R.id.sat -> day = getString(R.string.SAT)
-            -1 -> dayChips.check(wasCheckedDay)
+        val day = when (checkedId) {
+            R.id.sun -> getString(R.string.SUN)
+            R.id.mon -> getString(R.string.MON)
+            R.id.tue -> getString(R.string.TUE)
+            R.id.wed -> getString(R.string.WED)
+            R.id.thu -> getString(R.string.THU)
+            R.id.fri -> getString(R.string.FRI)
+            R.id.sat -> getString(R.string.SAT)
+            else -> FluxUtil.dayString
         }
         if (checkedId != -1 && wasCheckedDay != checkedId) {
             wasCheckedDay = checkedId
             selectedDay = day
-            val daysDifference = FluxUtil.getDayDifference(FluxUtil.dayString, selectedDay!!)
-            updateOperatingHoursOfSelectedDay(FluxUtil.getDateDaysAfter(daysDifference))
-        }
-    }
-
-    private fun setDayMenu(checkedId: Int) {
```

Comment:
ayyy ty for cleaning this up definitely was janky having both of those functions lool